.github/workflows/tests_windows.yml

```diff
@@ -38,6 +38,17 @@ jobs:
         with:
           arch: amd64
 
+      - name: Install lapack with vcpkg
+        run: |
+          pushd .
+          cd ~
+          git clone --quiet --recursive https://github.com/microsoft/vcpkg.git
+          cd vcpkg
+          .\bootstrap-vcpkg.bat
+          popd
+          vcpkg integrate install
```

Comment:
I'm not sure if this is required here. from the docs https://learn.microsoft.com/en-us/vcpkg/commands/integrate#vcpkg-integrate-install, this feature is embedded in VS17. 

```
On Windows with Visual Studio 2015, this subcommand will add redirecting logic into the MSBuild installation which will automatically pick up each user's user-wide vcpkg instance. Visual Studio 2017 and newer have this logic in the box.
```

Reply:
Let's remove it and see.